packages/icons/resources/icons/icons.json

```
@@ -3718,5 +3718,17 @@
         "iconName": "small-square",
         "tags": "square",
         "group": "miscellaneous"
+    },
+    {
+        "displayName": "Ship",
+        "iconName": "ship",
+        "tags": "transportation, ship, boat, cruise, yacht, move, supply, supply chain, trip, shipping",
+        "group": "interface"
+    },
+    {
+        "displayName": "Cargo Ship",
+        "iconName": "ship-cargo",
+        "tags": "transportation, cargo, cargo ship, freight, ship, boat, cruise, yacht, move, supply, supply chain, shipping container, shipping",
```

don't need "cargo ship" when both individual words exist in the tags list already

```suggestion
        "tags": "transportation, cargo, freight, ship, boat, cruise, yacht, supply, supply chain, container, shipping",
```